test/cookie.test.js

```diff
@@ -328,6 +328,32 @@ test('cookies signature', (t) => {
     })
   })
 
+  t.test('unsginCookie via fastify instance', t => {
+    t.plan(3)
+    const fastify = Fastify()
+    const secret = 'bar'
+
+    fastify.register(plugin, { secret })
+
+    fastify.get('/test1', (req, rep) => {
```

Comment:
Looks like it's my fault that the tests in this code base use arrow functions for handlers. In this case, I think it would be good to use a named or anonymous function such that `this.unsignCookie` is available.